<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>706</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>199</horstretch>
    <verstretch>199</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Potato Minus——管理员</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SIMS_res.qrc">
    <normaloff>:/SIM_System_Pics/resources/logo_3.png</normaloff>:/SIM_System_Pics/resources/logo_3.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1221</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
	border:none;
}
QTabWidget::tab-bar{
        alignment:left;
}
QTabBar::tab{
	background:transparent;
	color:black;
	min-width:30ex;
	min-height:10ex;
}
QTabBar::tab:hover{
	background:rgb(255, 255, 255, 100);
}
QTabBar::tab:selected{
	border-color: white;
	background:white;
	color:green;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="LessonTab">
     <attribute name="title">
      <string>课程管理</string>
     </attribute>
     <widget class="QTableWidget" name="LessonWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1221</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>111</number>
      </attribute>
      <column>
       <property name="text">
        <string>课程编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程教师</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程最大人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>当前选课人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课院系</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学分</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课年级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课学期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>上课时间</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="FlushButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ModifyLessonButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>修改课程信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="FindLessonButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>删除课程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="AddLessonButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>新增课程</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="TeacherTab">
     <attribute name="title">
      <string>教师管理</string>
     </attribute>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="FlushButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ModifyTeacherButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>修改教师信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="AddLessonButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>新增教师</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="DeleteTeacherButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>删除教师</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="TeacherWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>50</y>
        <width>1221</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>410</number>
      </attribute>
      <column>
       <property name="text">
        <string>教师编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>教师名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>所在院系</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="StudentTab">
     <attribute name="title">
      <string>学生管理</string>
     </attribute>
     <widget class="QTableWidget" name="StudentWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>50</y>
        <width>1221</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>243</number>
      </attribute>
      <column>
       <property name="text">
        <string>学生学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学生姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>性别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>所在院系</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>入学年级</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QToolButton" name="FlushButton_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ModifyStudentButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>修改学生信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="AddStudentButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>新增学生</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="DeleteStudentButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>删除学生</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ScholarTab">
     <attribute name="title">
      <string>奖学金管理</string>
     </attribute>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QToolButton" name="FlushButton_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_26">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>新增奖学金</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_27">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>删除奖学金</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="ScholarshipWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1221</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="gridSize">
       <size>
        <width>1000</width>
        <height>40</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>待处理事项</string>
     </attribute>
     <widget class="QListWidget" name="MsgWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>571</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="gridSize">
       <size>
        <width>1000</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>0</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/SIM_System_Pics/resources/logo3.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="MsgBoard">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>100</y>
      <width>301</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionLoadClassFromWeb"/>
    <addaction name="actionDeptManagement"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionInit"/>
    <addaction name="actionDatabaseRenew"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadClassFromWeb">
   <property name="text">
    <string>导入课程</string>
   </property>
  </action>
  <action name="actionInit">
   <property name="text">
    <string>数据库初始化</string>
   </property>
  </action>
  <action name="actionDeptManagement">
   <property name="text">
    <string>院系列表</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>程序信息</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>说明文档</string>
   </property>
  </action>
  <action name="actionDatabaseRenew">
   <property name="text">
    <string>重置数据库</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SIMS_res.qrc"/>
 </resources>
 <connections/>
</ui>
